components/alert/alert-toast.js

```diff
@@ -15,6 +15,7 @@ const states = {
 /**
  *  A component for communicating important information relating to the state of the system and the user's work flow, displayed as a pop-up at the bottom of the screen that automatically dismisses itself by default.
  * @slot - Default content placed inside of the component
+ * @fires d2l-alert-button-press - Dispatched when the alert's action button is clicked
```

Comment:
> ...but preventing the original event from bubbling may break existing usages and these would need to be changed

Could you do a quick audit and see if any of the places listening for this are listening on/from a `d2l-alert-toast`? If it's only a few places, then it might be cleanest to trap it and rename it as you said, but if it's a lot then maybe leaving it and documenting it is better.

Reply:
Yeah, it looks like it is only used in a single place, which I needed to change anyway so we can go with a `d2l-alert-toast-button-press` instead.

Reply:
Nice!

Reply:
So this will be updated to `d2l-alert-toast-button-press`?

Reply:
Yep, sorry I hadn't updated this PR yet since I'm still waiting on my PR in another repo to be merged. Fixed now!